### PR TITLE
View book details

### PR DESCRIPTION
**Pull request by hnamphamvo**
image
image

### CODE REVIEW

**Comment by hnamphamvo** [comment]
ready

File: app/views/books/show.html.erb [hunk 1]
```diff
@@ -1,2 +1,44 @@
...
+        </div>
+        <p class="product-description"><%= @book.description %></p>
+        <p class="product-description"><%= t("books.quantity_page") %><%= @book.number_of_pages %></p>
+        <div class="action">
+          <%= form_for(@book, url: books_path) do |f| %>
+            <%= f.number_field :number_of_pages, class: "form_num",
+              min: Settings.book.quantity_min,
+              max: Settings.book.quantity_max %>
+            <%= f.submit "add to cart", class: "add-to-cart btn btn-default", type: "button" %>
+          <% end %>
```
Comment (phamvanbk11) [inline app/views/books/show.html.erb]: tách form ra file riêng nhé

**Comment by phamvanbk11** [comment]
commented

**Comment by hnamphamvo** [comment]
ready

**Pull request merged**